### ISSUE TITLE
Import and Export PDF documents - Annotation

### ISSUE THREAD

**Issue by jonatap2chat** [muaz-khan/Canvas-Designer#43]
Hi muaz-khan,

I really like this tool, looks like just the thing I need to annotate documents, I just wondered if there is an option to import .pdf files, annotate and then export them?

I can see there is the import image and export image but I presume its possible to do the same with pdf or any other document extension potentially? 

As I really want to annotate something like a contact and then save the annotations and share.  Is this something you have done or a new feature I need to look into?

Many thanks

Jon

**Comment by ph4t0o0o0om** [comment]
Hey @jonatap2chat  do you have this feature? I badly need this. please help

**Comment by ph4t0o0o0om** [comment]
Bro please reply.

**Comment by muaz-khan** [comment]
What if we share & sync PDF separately? I mean, we can open a popup in the `index.html` demo to embed PDF and sync pages/zoom/etc. in realtime?

Please track this thread for PDF syncing: https://github.com/muaz-khan/RTCMultiConnection/issues/628#issuecomment-421596699

PS. If its easier then I'll embed PDF inside the main-canvas as well (same like images-tool).

**Comment by ph4t0o0o0om** [comment]
Yes sir that is what I'm going to do. To embed the pdf to canvas and can
draw to PDF.


Because I'm currently developing a VIRTUAL CLASSROOM sir. My inspiration is
https://www.braincert.com/try-virtual-classroom . that is what I'm going to
be.

And I using your library which is very helpful. Please help me Mr. muaz-khan

AGAIN THANK YOU SO MUCH!!! The muaz-khan/RTCMultiConnection#628 (comment)
<https://github.com/muaz-khan/RTCMultiConnection/issues/628#issuecomment-421596699>
that
is what I'm looking for.

**Comment by ph4t0o0o0om** [comment]
I'm going to wait for the upload feature mr muaz-khan
.

THANK YOU SO MUCH!

**Comment by ph4t0o0o0om** [comment]
Hi MR muaz-khan Good day!!!

When I'm going to see the PDF SYCNING function? I'm so very excited for
this!!


THANK YOU SO MUCH!!!
